integrationTests/testProcessorNodeWithCoordinator.go

```diff
@@ -101,17 +99,51 @@ func CreateProcessorNodesWithNodesCoordinator(
 				fmt.Println("error creating node coordinator")
 			}
 
-			tpn := newTestProcessorNodeWithCustomNodesCoordinator(
-				numShards,
-				shardId,
-				nodesCoordinatorInstance,
-				i,
-				ncp,
-				nodesSetup,
+			blsHasher, _ := blake2b.NewBlake2bWithSize(hashing.BlsHashSize)
+			llsig := &multisig2.BlsMultiSigner{Hasher: blsHasher}
+
+			pubKeysMap := pubKeysMapFromKeysMap(ncp)
+			kp := ncp[shardId][i]
+			multiSigner, err := multisig.NewBLSMultisig(
+				llsig,
+				pubKeysMap[shardId],
+				kp.BlockSignSk,
+				kp.BlockSignKeyGen,
+				uint16(i),
 			)
```

Comment:
can we use the `createMultiSigner` function from the other file (withMultiSigner) here also?

Reply:
done